api/Request.json

```diff
@@ -6,16 +6,16 @@
         "spec_url": "https://fetch.spec.whatwg.org/#request-class",
         "support": {
           "chrome": {
-            "version_added": "42"
+            "version_added": "40"
```

Comment:
This doesn't match the data for `fetch()` for reasons covered in https://github.com/mdn/browser-compat-data/issues/12340.

It would be great to keep things consistent, with 40 everywhere or 42 everywhere.

Reply:
Ah right, I forgot about that issue -- I think that having "40" everywhere makes more sense and tells a more accurate story, maybe with a note (or possibly a partial_implementation range) stating it's only available in service workers?